docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -1,69 +1,32 @@
 ---
-title: Run E2E tests on EAS Build
+title: Run E2E tests on EAS Workflows
```

Comment:
Let's create a sub-section under EAS Workflows for "Reference" and move this guide there.

Reply:
Do we need to create a redirect handler?

Reply:
Great point! Yes, we should add the redirect for the previous URL to the new one in both `error-utilities.ts` and `deploy.sh` files.